src/test/java/com/github/bot/curiosone/core/extraction/BrainResponseTest.java

```diff
@@ -1,5 +1,6 @@
 package com.github.bot.curiosone.core.extraction;
 
+import static org.assertj.core.api.Assertions.assertThat;
 // SUPPRESS CHECKSTYLE AvoidStarImport
 import static org.junit.Assert.*;
```

Comment:
I think you can remove this?

Reply:
I'm talking about this:
![image](https://user-images.githubusercontent.com/3505087/31057987-48f5621a-a6ec-11e7-9023-6b42455acae0.png)